scripts/ci-nvidia-device-plugin/tests/04-verify-device-plugin.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+if microshift --help | grep -q "healthcheck"; then
+    sudo microshift healthcheck \
+        --namespace nvidia-device-plugin \
+        --daemonsets nvidia-device-plugin-daemonset
+else
```

Comment:
Nit: feel free to fix in another PR. 

I think we need to add a comment here or in a README why we're checking it. The fact that these scripts will be used in multiple releases is not something we'd usually do